src/Module/Admin/Summary.php

```diff
@@ -79,7 +79,8 @@ protected function content(array $request = []): string
 		// Check if github.com/friendica/stable/VERSION is higher then
 		// the local version of Friendica. Check is opt-in, source may be stable or develop branch
 		if (DI::config()->get('system', 'check_new_version_url', 'none') != 'none') {
-			$gitversion = DI::config()->get('system', 'git_friendica_version');
+			$gitversion = DI::keyValue()->get('git_friendica_version');
```

Comment:
```suggestion
			$gitversion = DI::keyValue()->get('git_friendica_version') ?? ''; 
```

Reply:
Shouldn't both functions have got the same output when the requested parameter wasn't found?

Reply:
I'm guessing the original method call also needed a `?? ''` since we were already using the return value as a string.